Scoop/Model/Prompts.swift

```diff
@@ -14,7 +14,12 @@ struct Prompt: Codable {
     let answer: String?
 }
 
-struct UserAnswer: Codable {
+class UserAnswer: Codable {
     let id: Int
-    let answer: String
+    var answer: String
+    
+    init(id: Int, answer: String) {
+        self.id = id
+        self.answer = answer
+    }
```

Comment:
You're the developer so you understand the app more than I do, but is there a reason why you changed this to a class? Just curious.

Reply:
I'm passing it in as a reference type so that changes to the value apply during delegation, without having to pass a new value type each time